### PR TITLE
Update demo graphs (UI + documentation)

### PR DESCRIPTION
**Pull request by maxkernbach**
- all "knownGraphs" are now available in the UI
- documentation: added new graph viewer screenshots
- documentation: added "mps_graph" and some links

### CODE REVIEW

File: Documentation/Books/AQL/Examples/MultiplePaths.md [hunk 1]
```diff
@@ -2,9 +2,9 @@ Multiple Path Search
 ====================
 
 The shortest path algorithm can only determine one shortest path.
-For example, if this is the full graph:
+For example, if this is the full graph (based on the [mps_graph](../../Manual/Graphs/index.html#the-mps-graph)):
 
-![Example Graph](sp_graph.png)
+![Example Graph](../../Manual/Graphs/mps_graph.png)
```
Comment (dothebart) [inline Documentation/Books/AQL/Examples/MultiplePaths.md]: @Simran-B once this is merged and included, can you check back with pdf etc. whether it also picks these images properly? tnx.

**Comment by dothebart** [comment]
@Simran-B did you try out? afair we did the same before. We probably just didn't verify ;)

**Comment by Simran-B** [comment]
@dothebart 

I commented some code in build.sh, which failed in Cygwin (`/usr/bin/daemon` not found):

```sh
function build-dist-books()
{
    set -x
    set -e
    rm -rf books ppbooks
    PIDFILE=/tmp/xvfb_20_0.pid
    #if test -z "${DISPLAY}"; then
    #    DISPLAY=:20.0
    #    start_X11_display "${PIDFILE}" "${DISP}"
    #    trap 'stop_X11_display "${PIDFILE}"' 0
    #fi
    export DISPLAY
```

The resulting PDF and ePub files miss the graph picture.

PDF:

image

ePub:

image

**Comment by dothebart** [comment]
ok, are they here in existing pdfs of released documentation? or is this just an old error whe didn't notice before?

**Comment by dothebart** [comment]
re cygwin - you can workaround it by invoking `export DISPLAY=xxx` in advance. it should work, since java on the wintendo doesn't require an X11 display to render images.

**Comment by Simran-B** [comment]
@dothebart It's possible that we have such cross references elsewhere. Just found the following for example:

image

`../(Manual|AQL|HTTP|Cookbook|Drivers)/.*\.(png|jpg)`

```
AQL\Examples\CombiningGraphTraversals.md
AQL\Examples\MultiplePaths.md
HTTP\Gharial\Edges.md
HTTP\Gharial\Management.md
HTTP\Gharial\Vertices.md
```

File: Documentation/Books/Manual/Graphs/README.md [hunk 2]
```diff
@@ -280,6 +282,26 @@ It is used to demonstrate raw traversal operations.
     @END_EXAMPLE_ARANGOSH_RUN
     @endDocuBlock graph_create_world_sample
 
+### The Mps Graph
+
+This graph was created to demonstrate a use case of the shortest path algorithm. Even though the algorithm can only determine one shortest path, it is possible to return multiple shortest paths with two seperate queries. Therefore the graph is named after the [**m**(multiple)**p**(path)**s**(search)](../../AQL/Examples/MultiplePaths.html) use case.
```
Comment (Simran-B) [inline Documentation/Books/Manual/Graphs/README.md]: Typo: `seperate` --> `separate`

**Comment by dothebart** [comment]
@Simran-B please have a short look at my last commit to this branch - I dared to do it in @maxkernbach  s branch here to circumvent merge conflicts. The question arose in the community slack.

**Comment by sleto-it** [comment]
This PR contains changes to the UI code. I cannot merge it directly. I am adding Michael as a reviewer, and let him decide

**Pull request merged**